src/views/RecIM/components/Header/index.js

```diff
@@ -218,6 +170,10 @@ const Header = ({ page, expandable = false, match, team, activity, home, admin }
                   {match?.Team[0]?.Name ?? 'No team yet...'}
                 </Typography>
               </LinkRouter>
+              <Typography className={styles.subtitle}>
+                {/* once this is added to the API, it will instantly work */}
+                {match?.Team[0]?.TeamRecord.Win ?? 0}W : {match?.Team[0]?.TeamRecord.Loss ?? 0}L
+              </Typography>
```

Comment:
I viewed this as a non-admin, the header looks a bit naked without the Sportsmanship line. Added the W/L column just to fill negative space. Plus, it is a relatively nice feature. Feel free to adjust the styling. I just stole the subtitle style you had (obviously)